plugin/EstateList.php

```diff
@@ -936,15 +995,7 @@ public function getEstateContactIds(): array
 		$recordId = $this->_currentEstate['id'];
 		return $this->_estateContacts[$recordId] ?? [];
 	}
-	/**
-	 * @return bool
-	 */
-	public function isCurrentEstateContactsInAddressFilter()
```

Comment:
Ich denke wir müssen das drin lassen, falls individuelle Templates diese Methode nutzen. :(
Oder hast du eine Idee?

Reply:
glaube kaum, dass das jemand nutzt. 
Das hatten wir ja nur als Workaround für uns eingebaut und war ja nur 2-3 Wochen drin

Reply:
Es wird in pure verwendet:
https://github.com/onOffice-Web-Org/onoffice-pure/blob/135f35e8224bcacca788be9049340f7eafbd8cb9/onoffice-theme/templates/estate/property-card.php#L47

Reply:
Das ist bei einem Branch schon raus.
Also mit dem nächsten Release von Pure ist das auch raus.

Reply:
ist schon auf einem Branch gelöscht. Modern checke ich nochmal